flag_groups.go

```diff
@@ -23,9 +23,10 @@ import (
 )
 
 const (
-	requiredAsGroupAnnotation   = "cobra_annotation_required_if_others_set"
-	oneRequiredAnnotation       = "cobra_annotation_one_required"
-	mutuallyExclusiveAnnotation = "cobra_annotation_mutually_exclusive"
+	requiredAsGroupAnnotation             = "cobra_annotation_required_if_others_set"
+	oneRequiredAnnotation                 = "cobra_annotation_one_required"
+	mutuallyExclusiveAnnotation           = "cobra_annotation_mutually_exclusive"
+	ifPresentThenOthersRequiredAnnotation = "cobra_annotation_if_present_then_others_required"
```

Comment:
Could you rename them in a way they share a common prefix, not suffix

These constants are defined at the package level, so IDE will suggest them in any method when coding in that package

I would like to see something like this


```suggestion
	annotationGroupRequired             = "cobra_annotation_required_if_others_set"
	annotationRequiredOne                 = "cobra_annotation_one_required"
	annotationMutuallyExclusive           = "cobra_annotation_mutually_exclusive"
	annotationGroupDependent = "cobra_annotation_if_present_then_others_required"
```

The names of the constants are just suggestions to give you an idea